group.go

```diff
@@ -22,6 +24,32 @@ type GroupState struct {
 	AnyOn bool `json:"any_on,omitempty"`
 }
 
+// Stream define the stream status of a group
+type Stream struct {
+	ProxyMode string  `json:"proxymode,omitempty"`
+	ProxyNode string  `json:"proxynode,omitempty"`
+	ActiveRaw *bool   `json:"active,omitempty"`
+	OwnerRaw  *string `json:"owner,omitempty"`
```

Comment:
I don't think these need to be pointers. If they aren't then there shouldn't be a need for `Active()`and `Owner()` funcs since they will return init values.

Reply:
Owner is null when it's empty form the API itself.

Active requires the three states, true, false, and empty.
I discovered that there are some undesirable consequences of bool, and omitempty.

if active is not omitempty, you can accidently trigger changes to it from other calls that use the group struct.
The choice to use nil as the empty state comes from that.

nil ends up being the empty state that is omitted.

The DisableStreaming function will not work if Active is a bool with omitempty because false is the "empty" state.

Reply:
Ok, I understand. Sounds reasonable. Just make sure to have tests covering this. Thanks.